recipes/rdrs.rb

```diff
@@ -3,6 +3,18 @@
 conn_str_split = conn_str.split(/:/, 2)
 
 
+if node['ndb']['rdrs']['containerize'] == "true" 
+  bash 'Setting-RDRS-Image' do
+    user 'root'
```

Comment:
Use the `file_cache_path` as staging like we do at https://github.com/logicalclocks/ndb-chef/blob/master/recipes/install.rb#L179

Reply:
fixed